2022-03-18 16:39:35 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:39:35 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 16:39:35 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 16:39:35 [INFO] : --------------------------------------------
2022-03-18 16:39:35 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:39:35 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 16:39:36 [INFO] : 得到 [阿里云] 最新威胁情报 [2] 条
2022-03-18 16:39:36 [INFO] : --------------------------------------------
2022-03-18 16:39:36 [INFO] : 
==============================================
[ TITLE ] python python 未加控制的资源消耗（资源穷尽）
                        
[ TIME  ] 2022-03-05 16:39:36
[ CVE   ] CVE-2021-3737
[ SRC   ] 阿里云
[ URL   ] https://avd.aliyun.com/detail?id=AVD-2021-3737
2022-03-18 16:39:36 [INFO] : 
==============================================
[ TITLE ] Crestron Hd-Md4X2-4K-E授权错误漏洞（CVE-2022-23178）
                        
[ TIME  ] 2022-01-15 16:39:36
[ CVE   ] CVE-2022-23178
[ SRC   ] 阿里云
[ URL   ] https://avd.aliyun.com/detail?id=AVD-2022-23178
2022-03-18 16:39:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:39:36 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 16:39:57 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [10] 条
2022-03-18 16:39:57 [INFO] : --------------------------------------------
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] showdoc .webmv文件上传漏洞
[ TIME  ] 2022-03-18 16:39:37
[ CVE   ] CNVD-2022-20507 (CVE-2022-0964)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20507
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] showdoc .properties文件上传漏洞
[ TIME  ] 2022-03-18 16:39:39
[ CVE   ] CNVD-2022-20508 (CVE-2022-0966)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20508
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] showdoc .ofd文件上传漏洞
[ TIME  ] 2022-03-18 16:39:41
[ CVE   ] CNVD-2022-20509 (CVE-2022-0967)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20509
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] showdoc .md文件上传漏洞
[ TIME  ] 2022-03-18 16:39:43
[ CVE   ] CNVD-2022-20510 (CVE-2022-0956)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20510
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] showdoc .m3u8a文件上传漏洞
[ TIME  ] 2022-03-18 16:39:45
[ CVE   ] CNVD-2022-20511 (CVE-2022-0957)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20511
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] showdoc .cshtml文件上传漏洞
[ TIME  ] 2022-03-18 16:39:47
[ CVE   ] CNVD-2022-20512 (CVE-2022-0945)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20512
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] showdoc .aspx文件上传漏洞（ CNVD-2022-20513）
[ TIME  ] 2022-03-18 16:39:50
[ CVE   ] CNVD-2022-20513 (CVE-2022-0942)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20513
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] Microweber文件上传漏洞
[ TIME  ] 2022-03-18 16:39:52
[ CVE   ] CNVD-2022-20514 (CVE-2022-0963)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20514
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] Microweber跨站脚本漏洞
[ TIME  ] 2022-03-18 16:39:54
[ CVE   ] CNVD-2022-20515 (CVE-2022-0954)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20515
2022-03-18 16:39:57 [INFO] : 
==============================================
[ TITLE ] Microweber post title拒绝服务漏洞
[ TIME  ] 2022-03-18 16:39:56
[ CVE   ] CNVD-2022-20516 (CVE-2022-0961)
[ SRC   ] 国家信息安全漏洞共享平台（CNVD）
[ URL   ] https://www.cnvd.org.cn/flaw/show/CNVD-2022-20516
2022-03-18 16:39:57 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:39:57 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 16:40:12 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 16:40:12 [INFO] : --------------------------------------------
2022-03-18 16:40:12 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:40:12 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 16:40:17 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 16:40:17 [INFO] : --------------------------------------------
2022-03-18 16:40:17 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:40:17 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 16:40:19 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [6] 条
2022-03-18 16:40:19 [INFO] : --------------------------------------------
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] NATS nats-server before 2.7.4 allows Directory Traversal (with write access) via an element in a ZIP archive for JetStream streams. nats-streaming-server before 0.24.3 is also affected.
[ TIME  ] 2022-03-10 17:47:51 
[ CVE   ] CVE-2022-26652
[ SRC   ] 美国国家漏洞数据库（NVD）
[ URL   ] https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2022-26652
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] Abantecart through 1.3.2 allows remote authenticated administrators to execute arbitrary code by uploading an executable file, because the Catalog&gt;Media Manager&gt;Images settings can be changed by an administrator (e.g., by configuring .php to be a valid image file type).
[ TIME  ] 2022-03-10 17:47:46 
[ CVE   ] CVE-2022-26521
[ SRC   ] 美国国家漏洞数据库（NVD）
[ URL   ] https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2022-26521
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] Couchbase Operator 2.2.x before 2.2.3 exposes Sensitive Information to an Unauthorized Actor. Secrets are not redacted in logs collected from Kubernetes environments.
[ TIME  ] 2022-03-10 17:47:33 
[ CVE   ] CVE-2022-26311
[ SRC   ] 美国国家漏洞数据库（NVD）
[ URL   ] https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2022-26311
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] SAP Financial Consolidation - version 10.1, does not perform necessary authorization checks for updating homepage messages, resulting for an unauthorized user to alter the maintenance system message.
[ TIME  ] 2022-03-10 17:47:31 
[ CVE   ] CVE-2022-26104
[ SRC   ] 美国国家漏洞数据库（NVD）
[ URL   ] https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2022-26104
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] Under certain conditions, SAP NetWeaver (Real Time Messaging Framework) - version 7.50, allows an attacker to access information which could lead to information gathering for further exploits and attacks.
[ TIME  ] 2022-03-10 17:47:31 
[ CVE   ] CVE-2022-26103
[ SRC   ] 美国国家漏洞数据库（NVD）
[ URL   ] https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2022-26103
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] Due to missing authorization check, SAP NetWeaver Application Server for ABAP - versions 700, 701, 702, 731, allows an authenticated attacker, to access content on the start screen of any transaction that is available with in the same SAP system even if he/she isn't authorized for that transaction. A successful exploitation could expose information and in worst case manipulate data before the start screen is executed, resulting in limited impact on confidentiality and integrity of the application.
[ TIME  ] 2022-03-10 17:47:30 
[ CVE   ] CVE-2022-26102
[ SRC   ] 美国国家漏洞数据库（NVD）
[ URL   ] https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2022-26102
2022-03-18 16:40:19 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:40:19 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 16:40:19 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [10] 条
2022-03-18 16:40:19 [INFO] : --------------------------------------------
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] RESERVED CVE CANDIDATE NUMBER...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] CVE-2021-25220
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-383969
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] RESERVED CVE CANDIDATE NUMBER...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] CVE-2022-0485
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-414076
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] RESERVED CVE CANDIDATE NUMBER...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] CVE-2022-0843
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-416071
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] RESERVED CVE CANDIDATE NUMBER...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] CVE-2022-26383
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-417084
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] RESERVED CVE CANDIDATE NUMBER...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] CVE-2022-26384
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-417085
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] RESERVED CVE CANDIDATE NUMBER...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] CVE-2022-26387
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-417088
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] BuilderRevengeRAT XML Injection...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] VHN-417827
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-417827
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] BuilderTorCTPHPRAT.b Cross Site Scripting...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] VHN-417828
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-417828
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] BuilderTorCTPHPRAT.b Shell Upload...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] VHN-417829
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-417829
2022-03-18 16:40:19 [INFO] : 
==============================================
[ TITLE ] BuilderTorCTPHPRAT.b Insecure Credential Storage...
                    
[ TIME  ] 2022-03-17 16:40:19
[ CVE   ] VHN-417830
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-417830
2022-03-18 16:40:19 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:40:19 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 16:40:20 [INFO] : 得到 [绿盟] 最新威胁情报 [3] 条
2022-03-18 16:40:20 [INFO] : --------------------------------------------
2022-03-18 16:40:20 [INFO] : 
==============================================
[ TITLE ] Sourcecodester The Electric Billing Management System跨站脚本漏洞
[ TIME  ] 2022-03-18 16:40:20
[ CVE   ] CVE-2021-41929
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63628
2022-03-18 16:40:20 [INFO] : 
==============================================
[ TITLE ] StarWind Command Center权限提升漏洞
[ TIME  ] 2022-03-18 16:40:20
[ CVE   ] CVE-2022-23858
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63627
2022-03-18 16:40:20 [INFO] : 
==============================================
[ TITLE ] Ubuntu Cached-Path-Relative原型污染漏洞
[ TIME  ] 2022-03-18 16:40:20
[ CVE   ] CVE-2021-23518
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63626
2022-03-18 16:40:20 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:40:20 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 16:40:20 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 16:40:20 [INFO] : --------------------------------------------
2022-03-18 16:40:20 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:40:20 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 16:40:22 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 16:40:22 [INFO] : --------------------------------------------
2022-03-18 16:40:22 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:40:22 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 16:40:24 [INFO] : 得到 [斗象] 最新威胁情报 [1] 条
2022-03-18 16:40:24 [INFO] : --------------------------------------------
2022-03-18 16:40:24 [INFO] : 
==============================================
[ TITLE ] OpenSSL拒绝服务漏洞(CVE-2022-0778)
[ TIME  ] 2022-03-18 15:09:22
[ CVE   ] CVE-2022-0778
[ SRC   ] 斗象
[ URL   ] https://vas.riskivy.com/vuln-detail?id=115
2022-03-18 16:40:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:40:24 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 16:40:25 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [10] 条
2022-03-18 16:40:25 [INFO] : --------------------------------------------
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] Rapid7 Nexpose versions 6.6.129 and earlier suffer from a reflected cross site scripting vulnerability, within the shared scan configuration component of the tool. With this vulnerability an attacker could pass literal values as the test credentials, providing the opportunity for a potential XSS attack. This issue is fixed in Rapid7 Nexpose version 6.6.130.
[ TIME  ] 2022-03-17 23:15:00
[ CVE   ] CVE-2022-0758
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-0758
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] Rapid7 Nexpose versions 6.6.93 and earlier are susceptible to an SQL Injection vulnerability, whereby valid search operators are not defined. This lack of validation can allow an attacker to manipulate the "ANY" and "OR" operators in the SearchCriteria and inject SQL code. This issue was fixed in Rapid7 Nexpose version 6.6.129.
[ TIME  ] 2022-03-17 23:15:00
[ CVE   ] CVE-2022-0757
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-0757
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] Rapid7 Insight Agent versions 3.1.2.38 and earlier suffer from a privilege escalation vulnerability, whereby an attacker can hijack the flow of execution due to an unquoted argument to the runas.exe command used by the ir_agent.exe component, resulting in elevated rights and persistent access to the machine. This issue was fixed in Rapid7 Insight Agent version 3.1.3.80.
[ TIME  ] 2022-03-17 23:15:00
[ CVE   ] CVE-2022-0237
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-0237
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] In Paramiko before 2.10.1, a race condition (between creation and chmod) in the write_private_key_file function could allow unauthorized information disclosure.
[ TIME  ] 2022-03-17 22:15:00
[ CVE   ] CVE-2022-24302
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-24302
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] An SQL Injection vulnerability exists in Sourcecodester Attendance and Payroll System v1.0 which allows a remote attacker to bypass authentication via unsanitized login parameters.
[ TIME  ] 2022-03-17 22:15:00
[ CVE   ] CVE-2021-44088
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-44088
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] A Remote Code Execution (RCE) vulnerability exists in Sourcecodester Attendance and Payroll System v1.0 which allows an unauthenticated remote attacker to upload a maliciously crafted PHP via photo upload.
[ TIME  ] 2022-03-17 22:15:00
[ CVE   ] CVE-2021-44087
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-44087
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] Sonatype Nexus Repository Manager 3.36.0 allows HTML Injection.
[ TIME  ] 2022-03-17 22:15:00
[ CVE   ] CVE-2021-43961
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-43961
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] Improper authentication in Veeam Backup & Replication 9.5U3, 9.5U4,10.x and 11.x component used for Microsoft System Center Virtual Machine Manager (SCVMM) allows attackers execute arbitrary code via Veeam.Backup.PSManager.exe
[ TIME  ] 2022-03-17 21:15:00
[ CVE   ] CVE-2022-26504
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-26504
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] Veeam Backup & Replication 10.x and 11.x has Incorrect Access Control (issue 1 of 2).
[ TIME  ] 2022-03-17 21:15:00
[ CVE   ] CVE-2022-26501
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-26501
2022-03-18 16:40:25 [INFO] : 
==============================================
[ TITLE ] Improper limitation of path names in Veeam Backup & Replication 9.5U3, 9.5U4,10.x, and 11.x allows remote authenticated users access to internal API functions that allows attackers to upload and execute arbitrary code.
[ TIME  ] 2022-03-17 21:15:00
[ CVE   ] CVE-2022-26500
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-26500
2022-03-18 16:40:25 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 16:40:25 [INFO] : [邮件] 收件人清单： ['207176126@qq.com', '3074924881@qq.com']
2022-03-18 16:40:25 [ERROR] : [邮件] 推送威胁情报失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
2022-03-18 16:40:25 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
2022-03-18 16:40:25 [INFO] : 正在提交变更...
2022-03-18 16:40:26 [ERROR] : 提交变更失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'ssh_exchange_identification: read: Connection reset by peer
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.'
2022-03-18 16:40:26 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'ssh_exchange_identification: read: Connection reset by peer
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'ssh_exchange_identification: read: Connection reset by peer
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.'
2022-03-18 16:41:43 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:41:43 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 16:41:43 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 16:41:43 [INFO] : --------------------------------------------
2022-03-18 16:41:43 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:41:43 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 16:41:44 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 16:41:44 [INFO] : --------------------------------------------
2022-03-18 16:41:44 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:41:44 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 16:42:07 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 16:42:07 [INFO] : --------------------------------------------
2022-03-18 16:42:07 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:07 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 16:42:20 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 16:42:20 [INFO] : --------------------------------------------
2022-03-18 16:42:20 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:20 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 16:42:21 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 16:42:21 [INFO] : --------------------------------------------
2022-03-18 16:42:21 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:21 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 16:42:22 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 16:42:22 [INFO] : --------------------------------------------
2022-03-18 16:42:22 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:22 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 16:42:23 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 16:42:23 [INFO] : --------------------------------------------
2022-03-18 16:42:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:23 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 16:42:23 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 16:42:23 [INFO] : --------------------------------------------
2022-03-18 16:42:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:23 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 16:42:24 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 16:42:24 [INFO] : --------------------------------------------
2022-03-18 16:42:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:24 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 16:42:24 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 16:42:24 [INFO] : --------------------------------------------
2022-03-18 16:42:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:24 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 16:42:27 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 16:42:27 [INFO] : --------------------------------------------
2022-03-18 16:42:27 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:42:27 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 16:42:27 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 16:42:27 [INFO] : --------------------------------------------
2022-03-18 16:42:27 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 16:42:27 [INFO] : [邮件] 收件人清单： ['3074924881@qq.com', '207176126@qq.com']
2022-03-18 16:42:27 [ERROR] : [邮件] 推送威胁情报失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
2022-03-18 16:42:27 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
2022-03-18 16:42:27 [INFO] : 正在提交变更...
2022-03-18 16:42:34 [INFO] : 提交变更成功
2022-03-18 16:45:03 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:03 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 16:45:03 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 16:45:03 [INFO] : --------------------------------------------
2022-03-18 16:45:03 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:03 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 16:45:04 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 16:45:04 [INFO] : --------------------------------------------
2022-03-18 16:45:04 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:04 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 16:45:28 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 16:45:28 [INFO] : --------------------------------------------
2022-03-18 16:45:28 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:28 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 16:45:42 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 16:45:42 [INFO] : --------------------------------------------
2022-03-18 16:45:42 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:42 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 16:45:42 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 16:45:42 [INFO] : --------------------------------------------
2022-03-18 16:45:42 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:42 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 16:45:44 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 16:45:44 [INFO] : --------------------------------------------
2022-03-18 16:45:44 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:44 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 16:45:44 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 16:45:44 [INFO] : --------------------------------------------
2022-03-18 16:45:44 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:44 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 16:45:45 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 16:45:45 [INFO] : --------------------------------------------
2022-03-18 16:45:45 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:45 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 16:45:45 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 16:45:45 [INFO] : --------------------------------------------
2022-03-18 16:45:45 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:45 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 16:45:46 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 16:45:46 [INFO] : --------------------------------------------
2022-03-18 16:45:46 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:46 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 16:45:50 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 16:45:50 [INFO] : --------------------------------------------
2022-03-18 16:45:50 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:45:50 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 16:45:51 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 16:45:51 [INFO] : --------------------------------------------
2022-03-18 16:45:51 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 16:45:51 [INFO] : [邮件] 收件人清单： ['207176126@qq.com', '3074924881@qq.com']
2022-03-18 16:45:51 [ERROR] : [邮件] 推送威胁情报失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
2022-03-18 16:45:51 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
2022-03-18 16:45:51 [INFO] : 正在提交变更...
2022-03-18 16:45:51 [ERROR] : 提交变更失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'ssh_exchange_identification: read: Connection reset by peer
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.'
2022-03-18 16:45:51 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'ssh_exchange_identification: read: Connection reset by peer
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'ssh_exchange_identification: read: Connection reset by peer
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.'
2022-03-18 16:49:20 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:49:20 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 16:49:26 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 16:49:26 [INFO] : --------------------------------------------
2022-03-18 16:49:26 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:49:26 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 16:49:26 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 16:49:26 [INFO] : --------------------------------------------
2022-03-18 16:49:26 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:49:26 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 16:49:47 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 16:49:47 [INFO] : --------------------------------------------
2022-03-18 16:49:47 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:49:47 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 16:50:01 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 16:50:01 [INFO] : --------------------------------------------
2022-03-18 16:50:01 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:01 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 16:50:01 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 16:50:01 [INFO] : --------------------------------------------
2022-03-18 16:50:01 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:01 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 16:50:03 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 16:50:03 [INFO] : --------------------------------------------
2022-03-18 16:50:03 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:03 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 16:50:04 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 16:50:04 [INFO] : --------------------------------------------
2022-03-18 16:50:04 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:04 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 16:50:04 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 16:50:04 [INFO] : --------------------------------------------
2022-03-18 16:50:04 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:04 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 16:50:05 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 16:50:05 [INFO] : --------------------------------------------
2022-03-18 16:50:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:05 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 16:50:05 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 16:50:05 [INFO] : --------------------------------------------
2022-03-18 16:50:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:05 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 16:50:08 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 16:50:08 [INFO] : --------------------------------------------
2022-03-18 16:50:08 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:08 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 16:50:08 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 16:50:08 [INFO] : --------------------------------------------
2022-03-18 16:50:08 [INFO] : ('1024593536@qq.com', 'None')
2022-03-18 16:50:08 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 16:50:08 [INFO] : [邮件] 收件人清单： ['3074924881@qq.com', '207176126@qq.com']
2022-03-18 16:50:09 [ERROR] : [邮件] 推送威胁情报失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 47, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
2022-03-18 16:50:09 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 47, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 47, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 642, in auth
    authobject(challenge).encode('ascii'), eol='')
AttributeError: 'NoneType' object has no attribute 'encode'
2022-03-18 16:50:59 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:59 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 16:50:59 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 16:50:59 [INFO] : --------------------------------------------
2022-03-18 16:50:59 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:50:59 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 16:51:00 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 16:51:00 [INFO] : --------------------------------------------
2022-03-18 16:51:00 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:00 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 16:51:21 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 16:51:21 [INFO] : --------------------------------------------
2022-03-18 16:51:21 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:21 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 16:51:44 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 16:51:44 [INFO] : --------------------------------------------
2022-03-18 16:51:44 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:44 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 16:51:47 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 16:51:47 [INFO] : --------------------------------------------
2022-03-18 16:51:47 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:47 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 16:51:49 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 16:51:49 [INFO] : --------------------------------------------
2022-03-18 16:51:49 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:49 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 16:51:50 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 16:51:50 [INFO] : --------------------------------------------
2022-03-18 16:51:50 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:50 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 16:51:50 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 16:51:50 [INFO] : --------------------------------------------
2022-03-18 16:51:50 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:50 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 16:51:52 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 16:51:52 [INFO] : --------------------------------------------
2022-03-18 16:51:52 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:52 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 16:51:52 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 16:51:52 [INFO] : --------------------------------------------
2022-03-18 16:51:52 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:52 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 16:51:55 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 16:51:55 [INFO] : --------------------------------------------
2022-03-18 16:51:55 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 16:51:55 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 16:51:55 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 16:51:55 [INFO] : --------------------------------------------
2022-03-18 16:51:56 [INFO] : ('1024593536@qq.com', 'ntcrbdovjjdbbfde')
2022-03-18 16:51:56 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 16:51:56 [INFO] : [邮件] 收件人清单： ['207176126@qq.com', '3074924881@qq.com']
2022-03-18 16:51:57 [INFO] : [邮件] 推送威胁情报成功
2022-03-18 17:04:54 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:04:54 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 17:04:55 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 17:04:55 [INFO] : --------------------------------------------
2022-03-18 17:04:55 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:04:55 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 17:04:55 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 17:04:55 [INFO] : --------------------------------------------
2022-03-18 17:04:55 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:04:55 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 17:05:16 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 17:05:16 [INFO] : --------------------------------------------
2022-03-18 17:05:16 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:16 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 17:05:30 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 17:05:30 [INFO] : --------------------------------------------
2022-03-18 17:05:30 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:30 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 17:05:31 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 17:05:31 [INFO] : --------------------------------------------
2022-03-18 17:05:31 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:31 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 17:05:33 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 17:05:33 [INFO] : --------------------------------------------
2022-03-18 17:05:33 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:33 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 17:05:33 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 17:05:33 [INFO] : --------------------------------------------
2022-03-18 17:05:33 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:33 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 17:05:34 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 17:05:34 [INFO] : --------------------------------------------
2022-03-18 17:05:34 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:34 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 17:05:34 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 17:05:34 [INFO] : --------------------------------------------
2022-03-18 17:05:34 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:34 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 17:05:34 [INFO] : 得到 [红后] 最新威胁情报 [10] 条
2022-03-18 17:05:34 [INFO] : --------------------------------------------
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Aruba ClearPass Policy Manager Prior 命令注入漏洞
[ TIME  ] 2022-03-18 13:51:11
[ CVE   ] CVE-2021-26683
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=9784812dbc5c458880a766c1f80e874c
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Aruba ClearPass Policy Manager 操作系统命令注入漏洞
[ TIME  ] 2022-03-18 13:51:08
[ CVE   ] CVE-2021-26685
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=3e8e8a1d13144468acf3187d9606332f
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Aruba ClearPass Policy Manager 命令注入漏洞
[ TIME  ] 2022-03-18 13:51:04
[ CVE   ] CVE-2021-26680
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=325e02f8b9024ab19811981d1f0958f9
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Aruba ClearPass Policy Manager 跨站脚本漏洞
[ TIME  ] 2022-03-18 13:51:00
[ CVE   ] CVE-2021-26682
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=891985ab340548f9af40916737f882bb
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Aruba ClearPass Policy Manager Prior 跨站脚本漏洞
[ TIME  ] 2022-03-18 13:50:56
[ CVE   ] CVE-2021-26678
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=578433fbbf7147ccbbafae9dc27045fa
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Aruba ClearPass Policy Manager 命令注入漏洞
[ TIME  ] 2022-03-18 13:50:51
[ CVE   ] CVE-2021-26679
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=bb9f10b8183c4b7cbb26f8d4974d7a25
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Directus 安全漏洞
[ TIME  ] 2022-03-18 13:50:47
[ CVE   ] CVE-2021-26595
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=67484b98f89f45eb8f01892a56192d8d
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Directus 安全漏洞
[ TIME  ] 2022-03-18 13:50:43
[ CVE   ] CVE-2021-26594
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=b7a8cb26dd62492e800c58a131bf1dc4
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Apache Airflow 访问控制错误漏洞
[ TIME  ] 2022-03-18 13:50:39
[ CVE   ] CVE-2021-26559
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=4d31e72f050a41a98ee10fe6363f522a
2022-03-18 17:05:34 [INFO] : 
==============================================
[ TITLE ] Apache Livy 跨站脚本漏洞
[ TIME  ] 2022-03-18 13:50:35
[ CVE   ] CVE-2021-26544
[ SRC   ] 红后
[ URL   ] https://redqueen.tj-un.com/IntelDetails.html?id=c83710e48ca04571b5bfdf117e6a90e2
2022-03-18 17:05:34 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:34 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 17:05:37 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 17:05:37 [INFO] : --------------------------------------------
2022-03-18 17:05:37 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 17:05:37 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 17:05:38 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 17:05:38 [INFO] : --------------------------------------------
2022-03-18 17:05:38 [INFO] : ('1024593536@qq.com', 'ntcrbdovjjdbbfde')
2022-03-18 17:05:38 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 17:05:38 [INFO] : [邮件] 收件人清单： ['207176126@qq.com', '3074924881@qq.com']
2022-03-18 17:05:39 [INFO] : [邮件] 推送威胁情报成功
2022-03-18 22:27:10 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:10 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 22:27:11 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 22:27:11 [INFO] : --------------------------------------------
2022-03-18 22:27:11 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:11 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 22:27:11 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 22:27:11 [INFO] : --------------------------------------------
2022-03-18 22:27:11 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:11 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 22:27:33 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 22:27:33 [INFO] : --------------------------------------------
2022-03-18 22:27:33 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:33 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 22:27:49 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [10] 条
2022-03-18 22:27:49 [INFO] : --------------------------------------------
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] Rapid7 Nexpose 安全漏洞
[ TIME  ] 2022-03-17 22:27:34
[ CVE   ] CNNVD-202203-1607 (CVE-2022-0758)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1607
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] Rapid7 Nexpose 安全漏洞
[ TIME  ] 2022-03-17 22:27:35
[ CVE   ] CNNVD-202203-1606 (CVE-2022-0757)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1606
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] Rapid7 Insight Agent 安全漏洞
[ TIME  ] 2022-03-17 22:27:37
[ CVE   ] CNNVD-202203-1605 (CVE-2022-0237)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1605
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] Paramiko 安全漏洞
[ TIME  ] 2022-03-17 22:27:38
[ CVE   ] CNNVD-202203-1604 (CVE-2022-24302)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1604
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] Attendance and Payroll System 安全漏洞
[ TIME  ] 2022-03-17 22:27:40
[ CVE   ] CNNVD-202203-1603 (CVE-2021-44087)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1603
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] Attendance and Payroll System SQL注入漏洞
[ TIME  ] 2022-03-17 22:27:41
[ CVE   ] CNNVD-202203-1602 (CVE-2021-44088)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1602
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] Ligeo Archives 安全漏洞
[ TIME  ] 2022-03-17 22:27:43
[ CVE   ] CNNVD-202203-1601 (CVE-2021-46107)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1601
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] ljharb qs 安全漏洞
[ TIME  ] 2022-03-17 22:27:45
[ CVE   ] CNNVD-202203-1600 (CVE-2021-44907)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1600
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] NVIDIA FLARE 安全漏洞
[ TIME  ] 2022-03-17 22:27:46
[ CVE   ] CNNVD-202203-1599 (CVE-2022-21822)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1599
2022-03-18 22:27:49 [INFO] : 
==============================================
[ TITLE ] Gradio 安全漏洞
[ TIME  ] 2022-03-17 22:27:48
[ CVE   ] CNNVD-202203-1598 (CVE-2022-24770)
[ SRC   ] 国家信息安全漏洞库（CNNVD）
[ URL   ] http://www.cnnvd.org.cn/web/xxk/ldxqById.tag?CNNVD=CNNVD-202203-1598
2022-03-18 22:27:49 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:49 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 22:27:49 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 22:27:49 [INFO] : --------------------------------------------
2022-03-18 22:27:49 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:49 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 22:27:51 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [1] 条
2022-03-18 22:27:51 [INFO] : --------------------------------------------
2022-03-18 22:27:51 [INFO] : 
==============================================
[ TITLE ] Citrix Federated Authentication Service (FAS) 7.17 - 10.6 causes deployments that have been configured to store a registration authority certificate's private key in a Trusted Platform Module (TPM) to incorrectly store that key in the Microsoft Software Key Storage Provider (MSKSP). This issue only occurs if PowerShell was used when configuring FAS to store the registration authority certificate&acirc;&euro;&trade;s private key in the TPM. It does not occur if the TPM was not selected for use or if the FAS administration console was used for configuration.
[ TIME  ] 2022-03-10 17:47:36 
[ CVE   ] CVE-2022-26355
[ SRC   ] 美国国家漏洞数据库（NVD）
[ URL   ] https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2022-26355
2022-03-18 22:27:51 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:51 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 22:27:52 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [9] 条
2022-03-18 22:27:52 [INFO] : --------------------------------------------
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] guake 安全漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-23556
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-382017
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] Git 命令注入漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-23632
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-382093
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] notevil 注入漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-23771
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-382232
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] Attendance and Payroll System 安全漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-44087
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-406699
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] Attendance and Payroll System SQL注入漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-44088
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-406700
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] WAVLINK AC1200 安全漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-44259
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-407452
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] WAVLINK AC1200 安全漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-44260
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-407453
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] Netgear W104 安全漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-44261
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-407454
2022-03-18 22:27:52 [INFO] : 
==============================================
[ TITLE ] Netgear W104 安全漏洞
                    
[ TIME  ] 2022-03-17 22:27:52
[ CVE   ] CVE-2021-44262
[ SRC   ] 信息安全漏洞门户
[ URL   ] http://cve.scap.org.cn/vuln/VHN-407455
2022-03-18 22:27:52 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:52 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 22:27:56 [INFO] : 得到 [绿盟] 最新威胁情报 [15] 条
2022-03-18 22:27:56 [INFO] : --------------------------------------------
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Arris TR3300命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-27001
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63660
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Arris TR3300命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26999
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63659
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Arris TR3300命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-27002
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63658
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Arris Routers命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26994
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63657
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Arris TR3300命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26995
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63656
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Arris TR3300命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26996
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63655
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink多款路由器命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26208
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63654
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink多款路由器命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26207
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63653
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink多款路由器命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26209
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63652
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink多款路由器命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26210
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63651
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink多款路由器命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26211
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63650
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink多款路由器命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26212
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63649
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink X5000R命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26213
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63648
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink多款路由器命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-26206
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63647
2022-03-18 22:27:56 [INFO] : 
==============================================
[ TITLE ] Totolink Routers X5000R和A7000R命令注入漏洞
[ TIME  ] 2022-03-18 22:27:56
[ CVE   ] CVE-2022-27004
[ SRC   ] 绿盟
[ URL   ] http://www.nsfocus.net/vulndb/63646
2022-03-18 22:27:56 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:56 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 22:27:58 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 22:27:58 [INFO] : --------------------------------------------
2022-03-18 22:27:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:58 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 22:27:58 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 22:27:58 [INFO] : --------------------------------------------
2022-03-18 22:27:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:27:58 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 22:28:04 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 22:28:04 [INFO] : --------------------------------------------
2022-03-18 22:28:04 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:28:04 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 22:28:05 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [10] 条
2022-03-18 22:28:05 [INFO] : --------------------------------------------
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] A stack overflow vulnerability exists in the upnpd service in Netgear EX6100v1 201.0.2.28, CAX80 2.1.2.6, and DC112A 1.0.0.62, which may lead to the execution of arbitrary code without authentication.
[ TIME  ] 2022-03-18 11:15:00
[ CVE   ] CVE-2022-24655
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-24655
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] The Online Admission System 1.0 allows an unauthenticated attacker to upload or transfer files of dangerous types to the application through documents.php, which may be used to execute malicious code or lead to code execution.
[ TIME  ] 2022-03-18 11:15:00
[ CVE   ] CVE-2021-45835
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-45835
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] An attacker can upload or transfer files of dangerous types to the OpenDocMan 1.4.4 portal via add.php using MIME-bypass, which may be automatically processed within the product's environment or lead to arbitrary code execution.
[ TIME  ] 2022-03-18 11:15:00
[ CVE   ] CVE-2021-45834
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-45834
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] A local attacker could read files from some other users' SA360 reports stored in the /tmp folder during staging process before the files are loaded in BigQuery. We recommend upgrading to version 1.0.3 or above.
[ TIME  ] 2022-03-18 11:15:00
[ CVE   ] CVE-2021-22571
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-22571
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] golang.org/x/crypto/ssh before 0.0.0-20220314234659-1baeb1ce4c0b in Go through 1.16.15 and 1.17.x through 1.17.8 allows an attacker to crash a server in certain circumstances involving AddHostKey.
[ TIME  ] 2022-03-18 07:15:00
[ CVE   ] CVE-2022-27191
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-27191
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] In Pluck 4.7.16, an admin user can use the theme upload functionality at /admin.php?action=themeinstall to perform remote code execution.
[ TIME  ] 2022-03-18 07:15:00
[ CVE   ] CVE-2022-26965
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-26965
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] In the Linux kernel before 5.15.3, fs/quota/quota_tree.c does not validate the block number in the quota tree (on disk). This can, for example, lead to a kernel/locking/rwsem.c use-after-free if there is a corrupted quota file.
[ TIME  ] 2022-03-18 07:15:00
[ CVE   ] CVE-2021-45868
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-45868
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] scheme/webauthn.c in Glewlwyd SSO server 2.x before 2.6.2 has a buffer overflow associated with a webauthn assertion.
[ TIME  ] 2022-03-18 06:15:00
[ CVE   ] CVE-2022-27240
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2022-27240
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] An issue was discovered in xmppserver jar in the XMPP Server component of the JIve platform, as used in Pascom Cloud Phone System before 7.20.x (and in other products). An endpoint in the backend Tomcat server of the Pascom allows SSRF, a related issue to CVE-2019-18394.
[ TIME  ] 2022-03-18 05:15:00
[ CVE   ] CVE-2021-45968
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-45968
2022-03-18 22:28:05 [INFO] : 
==============================================
[ TITLE ] An issue was discovered in Pascom Cloud Phone System before 7.20.x. A configuration error between NGINX and a backend Tomcat server leads to a path traversal in the Tomcat server, exposing unintended endpoints.
[ TIME  ] 2022-03-18 05:15:00
[ CVE   ] CVE-2021-45967
[ SRC   ] Tenable (Nessus)
[ URL   ] https://www.tenable.com/cve/CVE-2021-45967
2022-03-18 22:28:05 [INFO] : ('1024593536@qq.com', 'ntcrbdovjjdbbfde')
2022-03-18 22:28:05 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 22:28:05 [INFO] : [邮件] 收件人清单： ['3074924881@qq.com', '207176126@qq.com']
2022-03-18 22:28:05 [INFO] : [邮件] 推送威胁情报成功
2022-03-18 22:28:05 [INFO] : 正在提交变更...
2022-03-18 22:28:10 [INFO] : 提交变更成功
2022-03-18 22:40:22 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:40:22 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 22:40:23 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 22:40:23 [INFO] : --------------------------------------------
2022-03-18 22:40:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:40:23 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 22:40:23 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 22:40:23 [INFO] : --------------------------------------------
2022-03-18 22:40:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:40:23 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 22:40:45 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 22:40:45 [INFO] : --------------------------------------------
2022-03-18 22:40:45 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:40:45 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 22:41:02 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 22:41:02 [INFO] : --------------------------------------------
2022-03-18 22:41:02 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:41:02 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 22:41:03 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 22:41:03 [INFO] : --------------------------------------------
2022-03-18 22:41:03 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:41:03 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 22:41:05 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 22:41:05 [INFO] : --------------------------------------------
2022-03-18 22:41:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:41:05 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 22:41:05 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 22:41:05 [INFO] : --------------------------------------------
2022-03-18 22:41:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:41:05 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 22:41:06 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 22:41:06 [INFO] : --------------------------------------------
2022-03-18 22:41:06 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:41:06 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 22:41:07 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 22:41:07 [INFO] : --------------------------------------------
2022-03-18 22:41:07 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:41:07 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 22:41:07 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 22:41:07 [INFO] : --------------------------------------------
2022-03-18 22:41:07 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:41:07 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 22:41:10 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 22:41:10 [INFO] : --------------------------------------------
2022-03-18 22:41:10 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:41:10 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 22:41:10 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 22:41:10 [INFO] : --------------------------------------------
2022-03-18 22:41:10 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 22:41:10 [INFO] : [邮件] 收件人清单： ['3074924881@qq.com', '207176126@qq.com']
2022-03-18 22:41:11 [ERROR] : [邮件] 推送威胁情报失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 734, in login
    raise last_exception
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 646, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'Login Fail. Please enter your authorization code to login. More information in http://service.mail.qq.com/cgi-bin/help?subtype=1&&id=28&&no=1001256')
2022-03-18 22:41:11 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 734, in login
    raise last_exception
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 646, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'Login Fail. Please enter your authorization code to login. More information in http://service.mail.qq.com/cgi-bin/help?subtype=1&&id=28&&no=1001256')
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 46, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 734, in login
    raise last_exception
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 646, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'Login Fail. Please enter your authorization code to login. More information in http://service.mail.qq.com/cgi-bin/help?subtype=1&&id=28&&no=1001256')
2022-03-18 22:41:11 [INFO] : 正在提交变更...
2022-03-18 22:41:14 [ERROR] : 提交变更失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first merge the remote changes (e.g.,
hint: 'git pull') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
2022-03-18 22:41:14 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first merge the remote changes (e.g.,
hint: 'git pull') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first merge the remote changes (e.g.,
hint: 'git pull') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
2022-03-18 22:51:35 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:51:35 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 22:51:35 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 22:51:35 [INFO] : --------------------------------------------
2022-03-18 22:51:35 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:51:35 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 22:51:36 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 22:51:36 [INFO] : --------------------------------------------
2022-03-18 22:51:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:51:36 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 22:51:57 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 22:51:57 [INFO] : --------------------------------------------
2022-03-18 22:51:57 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:51:57 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 22:52:16 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 22:52:16 [INFO] : --------------------------------------------
2022-03-18 22:52:16 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:52:16 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 22:52:16 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 22:52:16 [INFO] : --------------------------------------------
2022-03-18 22:52:16 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:52:16 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 22:52:18 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 22:52:18 [INFO] : --------------------------------------------
2022-03-18 22:52:18 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:52:18 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 22:52:20 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 22:52:20 [INFO] : --------------------------------------------
2022-03-18 22:52:20 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:52:20 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 22:52:24 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 22:52:24 [INFO] : --------------------------------------------
2022-03-18 22:52:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:52:24 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 22:52:24 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 22:52:24 [INFO] : --------------------------------------------
2022-03-18 22:52:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:52:24 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 22:52:24 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 22:52:24 [INFO] : --------------------------------------------
2022-03-18 22:52:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:52:24 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 22:52:27 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 22:52:27 [INFO] : --------------------------------------------
2022-03-18 22:52:27 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:52:27 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 22:52:28 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 22:52:28 [INFO] : --------------------------------------------
2022-03-18 22:52:28 [INFO] : ('1024593536@qq.com', '${MAIL_PASS}')
2022-03-18 22:52:28 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 22:52:28 [INFO] : [邮件] 收件人清单： ['207176126@qq.com', '3074924881@qq.com']
2022-03-18 22:52:28 [ERROR] : [邮件] 推送威胁情报失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 47, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 734, in login
    raise last_exception
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 646, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'Login Fail. Please enter your authorization code to login. More information in http://service.mail.qq.com/cgi-bin/help?subtype=1&&id=28&&no=1001256')
2022-03-18 22:52:28 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 47, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 734, in login
    raise last_exception
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 646, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'Login Fail. Please enter your authorization code to login. More information in http://service.mail.qq.com/cgi-bin/help?subtype=1&&id=28&&no=1001256')
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/notice/mail.py", line 47, in to_mail
    smtpObj.login(sender, password)
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 734, in login
    raise last_exception
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 723, in login
    (code, resp) = self.auth(
  File "/usr/local/python3/lib/python3.8/smtplib.py", line 646, in auth
    raise SMTPAuthenticationError(code, resp)
smtplib.SMTPAuthenticationError: (535, b'Login Fail. Please enter your authorization code to login. More information in http://service.mail.qq.com/cgi-bin/help?subtype=1&&id=28&&no=1001256')
2022-03-18 22:54:01 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:01 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 22:54:01 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 22:54:01 [INFO] : --------------------------------------------
2022-03-18 22:54:01 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:01 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 22:54:02 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 22:54:02 [INFO] : --------------------------------------------
2022-03-18 22:54:02 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:02 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 22:54:23 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 22:54:23 [INFO] : --------------------------------------------
2022-03-18 22:54:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:23 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 22:54:38 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 22:54:38 [INFO] : --------------------------------------------
2022-03-18 22:54:38 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:38 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 22:54:40 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 22:54:40 [INFO] : --------------------------------------------
2022-03-18 22:54:40 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:40 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 22:54:43 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 22:54:43 [INFO] : --------------------------------------------
2022-03-18 22:54:43 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:43 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 22:54:45 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 22:54:45 [INFO] : --------------------------------------------
2022-03-18 22:54:45 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:45 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 22:54:46 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 22:54:46 [INFO] : --------------------------------------------
2022-03-18 22:54:46 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:46 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 22:54:47 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 22:54:47 [INFO] : --------------------------------------------
2022-03-18 22:54:47 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:47 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 22:54:47 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 22:54:47 [INFO] : --------------------------------------------
2022-03-18 22:54:47 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:47 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 22:54:51 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 22:54:51 [INFO] : --------------------------------------------
2022-03-18 22:54:51 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:54:51 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 22:54:52 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 22:54:52 [INFO] : --------------------------------------------
2022-03-18 22:54:52 [INFO] : ('1024593536@qq.com', 'ntcrbdovjjdbbfde')
2022-03-18 22:54:52 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 22:54:52 [INFO] : [邮件] 收件人清单： ['3074924881@qq.com', '207176126@qq.com']
2022-03-18 22:54:53 [INFO] : [邮件] 推送威胁情报成功
2022-03-18 22:56:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:56:58 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 22:56:59 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 22:56:59 [INFO] : --------------------------------------------
2022-03-18 22:56:59 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:56:59 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 22:57:00 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 22:57:00 [INFO] : --------------------------------------------
2022-03-18 22:57:00 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:00 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 22:57:22 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 22:57:22 [INFO] : --------------------------------------------
2022-03-18 22:57:22 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:22 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 22:57:41 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 22:57:41 [INFO] : --------------------------------------------
2022-03-18 22:57:41 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:41 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 22:57:41 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 22:57:41 [INFO] : --------------------------------------------
2022-03-18 22:57:41 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:41 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 22:57:43 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 22:57:43 [INFO] : --------------------------------------------
2022-03-18 22:57:43 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:43 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 22:57:43 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 22:57:43 [INFO] : --------------------------------------------
2022-03-18 22:57:43 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:43 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 22:57:52 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 22:57:52 [INFO] : --------------------------------------------
2022-03-18 22:57:52 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:52 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 22:57:53 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 22:57:53 [INFO] : --------------------------------------------
2022-03-18 22:57:53 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:53 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 22:57:53 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 22:57:53 [INFO] : --------------------------------------------
2022-03-18 22:57:53 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:53 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 22:57:56 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 22:57:56 [INFO] : --------------------------------------------
2022-03-18 22:57:56 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:57:56 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 22:57:57 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 22:57:57 [INFO] : --------------------------------------------
2022-03-18 22:57:57 [INFO] : ('1024593536@qq.com', 'ntcrbdovjjdbbfde')
2022-03-18 22:57:57 [INFO] : [邮件] 正在推送威胁情报...
2022-03-18 22:57:57 [INFO] : [邮件] 收件人清单： ['207176126@qq.com']
2022-03-18 22:57:57 [INFO] : [邮件] 推送威胁情报成功
2022-03-18 22:59:12 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:12 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 22:59:12 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 22:59:12 [INFO] : --------------------------------------------
2022-03-18 22:59:12 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:12 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 22:59:13 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 22:59:13 [INFO] : --------------------------------------------
2022-03-18 22:59:13 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:13 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 22:59:35 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 22:59:35 [INFO] : --------------------------------------------
2022-03-18 22:59:35 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:35 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 22:59:52 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 22:59:52 [INFO] : --------------------------------------------
2022-03-18 22:59:52 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:52 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 22:59:53 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 22:59:53 [INFO] : --------------------------------------------
2022-03-18 22:59:53 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:53 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 22:59:56 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 22:59:56 [INFO] : --------------------------------------------
2022-03-18 22:59:56 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:56 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 22:59:57 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 22:59:57 [INFO] : --------------------------------------------
2022-03-18 22:59:57 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:57 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 22:59:58 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 22:59:58 [INFO] : --------------------------------------------
2022-03-18 22:59:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:58 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 22:59:58 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 22:59:58 [INFO] : --------------------------------------------
2022-03-18 22:59:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:58 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 22:59:59 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 22:59:59 [INFO] : --------------------------------------------
2022-03-18 22:59:59 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 22:59:59 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 23:00:02 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 23:00:02 [INFO] : --------------------------------------------
2022-03-18 23:00:02 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:00:02 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 23:00:02 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 23:00:02 [INFO] : --------------------------------------------
2022-03-18 23:00:02 [INFO] : 正在提交变更...
2022-03-18 23:00:05 [ERROR] : 提交变更失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first merge the remote changes (e.g.,
hint: 'git pull') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
2022-03-18 23:00:05 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first merge the remote changes (e.g.,
hint: 'git pull') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first merge the remote changes (e.g.,
hint: 'git pull') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
2022-03-18 23:02:27 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:02:27 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 23:02:28 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 23:02:28 [INFO] : --------------------------------------------
2022-03-18 23:02:28 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:02:28 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 23:02:28 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 23:02:28 [INFO] : --------------------------------------------
2022-03-18 23:02:28 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:02:28 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 23:03:55 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 23:03:55 [INFO] : --------------------------------------------
2022-03-18 23:03:55 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:03:55 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 23:04:14 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 23:04:14 [INFO] : --------------------------------------------
2022-03-18 23:04:14 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:04:14 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 23:05:15 [ERROR] : 获取 [360 网络安全响应中心] 威胁情报异常
Traceback (most recent call last):
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/usr/local/python3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/usr/local/python3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/usr/local/python3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1091: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/python3/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='cert.360.cn', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/crawler/_base_crawler.py", line 62, in cves
    new_cves = self.get_cves()
  File "/root/project/threat-broadcast-1/src/crawler/cert360.py", line 48, in get_cves
    response = requests.get(
  File "/usr/local/python3/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='cert.360.cn', port=443): Read timed out. (read timeout=60)
2022-03-18 23:05:15 [ERROR] : Traceback (most recent call last):
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/usr/local/python3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/usr/local/python3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/usr/local/python3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1091: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/python3/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='cert.360.cn', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/crawler/_base_crawler.py", line 62, in cves
    new_cves = self.get_cves()
  File "/root/project/threat-broadcast-1/src/crawler/cert360.py", line 48, in get_cves
    response = requests.get(
  File "/usr/local/python3/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='cert.360.cn', port=443): Read timed out. (read timeout=60)
Traceback (most recent call last):
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/usr/local/python3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/usr/local/python3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/usr/local/python3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1091: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/python3/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/usr/local/python3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='cert.360.cn', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/crawler/_base_crawler.py", line 62, in cves
    new_cves = self.get_cves()
  File "/root/project/threat-broadcast-1/src/crawler/cert360.py", line 48, in get_cves
    response = requests.get(
  File "/usr/local/python3/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='cert.360.cn', port=443): Read timed out. (read timeout=60)
2022-03-18 23:05:15 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 23:05:15 [INFO] : --------------------------------------------
2022-03-18 23:05:15 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:05:15 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 23:05:18 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 23:05:18 [INFO] : --------------------------------------------
2022-03-18 23:05:18 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:05:18 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 23:05:19 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 23:05:19 [INFO] : --------------------------------------------
2022-03-18 23:05:19 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:05:19 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 23:05:19 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 23:05:19 [INFO] : --------------------------------------------
2022-03-18 23:05:19 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:05:19 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 23:05:20 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 23:05:20 [INFO] : --------------------------------------------
2022-03-18 23:05:20 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:05:20 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 23:05:21 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 23:05:21 [INFO] : --------------------------------------------
2022-03-18 23:05:21 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:05:21 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 23:05:24 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 23:05:24 [INFO] : --------------------------------------------
2022-03-18 23:05:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:05:24 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 23:05:25 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 23:05:25 [INFO] : --------------------------------------------
2022-03-18 23:05:25 [INFO] : 正在提交变更...
2022-03-18 23:05:28 [ERROR] : 提交变更失败
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Merge the remote changes (e.g. 'git pull')
hint: before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
2022-03-18 23:05:28 [ERROR] : Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Merge the remote changes (e.g. 'git pull')
hint: before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
Traceback (most recent call last):
  File "/root/project/threat-broadcast-1/src/utils/_git.py", line 26, in auto_commit
    repo.git.push()
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 542, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 1005, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/usr/local/python3/lib/python3.8/site-packages/git/cmd.py", line 822, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To git@github.com:hkmayfly/threat-broadcast-1.git
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'git@github.com:hkmayfly/threat-broadcast-1.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Merge the remote changes (e.g. 'git pull')
hint: before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
2022-03-18 23:11:35 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:11:35 [INFO] : 正在获取 [数字观星POC++] 威胁情报...
2022-03-18 23:11:35 [INFO] : 得到 [数字观星POC++] 最新威胁情报 [0] 条
2022-03-18 23:11:35 [INFO] : --------------------------------------------
2022-03-18 23:11:35 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:11:35 [INFO] : 正在获取 [阿里云] 威胁情报...
2022-03-18 23:11:36 [INFO] : 得到 [阿里云] 最新威胁情报 [0] 条
2022-03-18 23:11:36 [INFO] : --------------------------------------------
2022-03-18 23:11:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:11:36 [INFO] : 正在获取 [国家信息安全漏洞共享平台（CNVD）] 威胁情报...
2022-03-18 23:11:58 [INFO] : 得到 [国家信息安全漏洞共享平台（CNVD）] 最新威胁情报 [0] 条
2022-03-18 23:11:58 [INFO] : --------------------------------------------
2022-03-18 23:11:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:11:58 [INFO] : 正在获取 [国家信息安全漏洞库（CNNVD）] 威胁情报...
2022-03-18 23:12:14 [INFO] : 得到 [国家信息安全漏洞库（CNNVD）] 最新威胁情报 [0] 条
2022-03-18 23:12:14 [INFO] : --------------------------------------------
2022-03-18 23:12:14 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:12:14 [INFO] : 正在获取 [360 网络安全响应中心] 威胁情报...
2022-03-18 23:12:15 [INFO] : 得到 [360 网络安全响应中心] 最新威胁情报 [0] 条
2022-03-18 23:12:15 [INFO] : --------------------------------------------
2022-03-18 23:12:15 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:12:15 [INFO] : 正在获取 [美国国家漏洞数据库（NVD）] 威胁情报...
2022-03-18 23:12:17 [INFO] : 得到 [美国国家漏洞数据库（NVD）] 最新威胁情报 [0] 条
2022-03-18 23:12:17 [INFO] : --------------------------------------------
2022-03-18 23:12:17 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:12:17 [INFO] : 正在获取 [信息安全漏洞门户] 威胁情报...
2022-03-18 23:12:17 [INFO] : 得到 [信息安全漏洞门户] 最新威胁情报 [0] 条
2022-03-18 23:12:17 [INFO] : --------------------------------------------
2022-03-18 23:12:17 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:12:17 [INFO] : 正在获取 [绿盟] 威胁情报...
2022-03-18 23:12:17 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2022-03-18 23:12:17 [INFO] : --------------------------------------------
2022-03-18 23:12:17 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:12:17 [INFO] : 正在获取 [奇安信] 威胁情报...
2022-03-18 23:12:18 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2022-03-18 23:12:18 [INFO] : --------------------------------------------
2022-03-18 23:12:18 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:12:18 [INFO] : 正在获取 [红后] 威胁情报...
2022-03-18 23:12:18 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2022-03-18 23:12:18 [INFO] : --------------------------------------------
2022-03-18 23:12:18 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:12:18 [INFO] : 正在获取 [斗象] 威胁情报...
2022-03-18 23:12:24 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2022-03-18 23:12:24 [INFO] : --------------------------------------------
2022-03-18 23:12:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2022-03-18 23:12:24 [INFO] : 正在获取 [Tenable (Nessus)] 威胁情报...
2022-03-18 23:12:24 [INFO] : 得到 [Tenable (Nessus)] 最新威胁情报 [0] 条
2022-03-18 23:12:24 [INFO] : --------------------------------------------
2022-03-18 23:12:24 [INFO] : 正在提交变更...
